src/_about/faq.md

```diff
@@ -0,0 +1,142 @@
+due to concerns around governance, project management and moderation. You can read more about the Quilt project and 
+where it came from on [the About Us page](/about).
+
+{% endadmonition %}
+{% admonition %}
+
+## Which parts of Fabric is Quilt forking?
+
+All of them. Quilt is a hard fork of Fabric, and we aim to fork and maintain all projects necessary to exist
+independently of it.
+
+{% endadmonition %}
+{% admonition %}
+
+## Why should I use Quilt?
+
+For users, Quilt provides the following advantages:
+
+* Mods that should be more compatible with each other, despite their use of powerful modding tools
+* Automatic downloading of Quilt's libraries (which can be disabled), so you always have exactly - and only - what you 
+  need for your mods to work
+* Better errors when your mod list has problems with missing or incompatible mods, with more friendly instructions
+* Initial compatibility with both Fabric and Quilt mods, meaning you don't need to stop using your favourites
+* An inclusive, transparent and friendly official community that cares about social issues, has an experienced staff 
+  team, and works with its users to improve over time
+
+{% endadmonition %}
+{% admonition %}
+
+## How can I support Quilt?
+
+If you'd like to help us out, here's just a few ways we've come up with:
+
+* Join [our community spaces](/community), hang out and give your input
+* Contribute to [our projects on GitHub](https://github.com/quiltmc), either directly or by providing reviews and
+  opinions
+* Support other projects that make use of Quilt, or Quilt's tools and libraries
+* Spread the good word on social media (in a responsible manner)
+
+[//]: # (* Help with ongoing costs by [supporting us financially on OpenCollective]&#40;https://opencollective.com/quiltmc&#41;)
+
+Ultimately, even simply making use of Quilt or hanging out in our community spaces helps us. All activity, input and
+contributions are appreciated!
+
+{% endadmonition %}
+
+# Usage Questions {#usage-questions}
+
+{% admonition %}
+
+## Can Quilt load Forge mods?
+
+No, Quilt can't load Forge mods and support for this is not planned. There are third-party projects (such as Patchwork)
+that plan to add this functionality, but they're not maintained by Quilt - and they're not part of The Quilt Project.
+
+{% endadmonition %}
+{% admonition %}
+
+## Can Quilt load Fabric mods?
+
+In most cases, yes. As Quilt is a fork, most Fabric mods should be compatible with Quilt initially - but this may not be
+the case forever. Once Quilt is properly-established, we do plan on dropping first-party support for Fabric mods. That
+said, there's no reason the community can't continue to maintain that support, and we'll provide the required resources
+if someone else decides to continue that part of the project after we stop maintaining it.
+
+It's also worth noting that - as of this writing - Fabric mods that contain conditional mixins are not supported. This
```

Comment:
See my comment on this section in faq_dev--as of writing this still works, and the issue is mixin plugins, not conditional mixins.
Lack of support for mixin plugins also means that mods using Fabric ASM will not work.